test/src/test/java/org/corfudb/integration/SecurityIT.java

```diff
@@ -0,0 +1,220 @@
+package org.corfudb.integration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.google.common.reflect.TypeToken;
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.collections.CorfuTable;
+import org.corfudb.runtime.exceptions.unrecoverable.UnrecoverableCorfuError;
+import org.corfudb.util.NodeLocator;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+
+/**
+ * This test suit exercises the ability to enable TLS on Corfu servers and runtime
+ *
+ * Created by Sam Behnam on 8/13/18.
+ */
+@Slf4j
+public class SecurityIT extends AbstractIT {
+    private String corfuSingleNodeHost;
```

Comment:
This is duplicate code. Should we move it to AbstractIT?

Reply:
Although there are properties such as host and port that SecurityIT reuses, I want to keep the code retrieving them separate as they can be different/changed for each test. For instance, this test has its own specific tlsEnable properties.